src/components/TableDisplay/TableDisplayTs.ts

```diff
@@ -258,7 +268,7 @@ export class TableDisplayTs extends Vue {
     }
 
     const mosaics = await this.$store.dispatch('wallet/REST_FETCH_OWNED_MOSAICS', this.currentWalletAddress.plain())
-    new MosaicService(this.$store).refreshMosaicModels(mosaics, true)
+    await new MosaicService(this.$store).refreshMosaicModels(mosaics, true)
```

Comment:
I am not convinced about this change ; can you explain why you need those blocking requests?

Reply:
In order to prevent frequent refreshes, if this is not good, I will fix it.

Reply:
The "await"  was removed